static_report/src/components/SingleReport/SRTableList/SRTableListColumnDetail.tsx

```diff
@@ -0,0 +1,54 @@
+import { Flex, Grid, GridItem, Text } from '@chakra-ui/react';
+
+import { ColumnName } from '../../shared/TableList/ColumnName';
+import { SRTableListAssertionsSummary } from './SRTableListAssertionsSummary';
+import { HistogramChart } from '../../shared/Charts/HistogramChart';
+import type {
+  AssertionTest,
+  ColumnSchema,
+} from '../../../sdlc/single-report-schema';
+
+export function SRTableListColumnDetail({
+  name,
+  data,
+  icon,
+  colAssertions,
+}: {
+  name: string;
+  data: ColumnSchema;
```

Comment:
suggestion: `columnDatum`

let's try and reserve `data` for the high-level consumption of `*ReportSchema`.
Everything downstream should be a bit more specific, naming wise, for clarity

Reply:
done